### PR TITLE
Strict-Transport-Security for redirects

### PR DESCRIPTION
**Pull request by szepeviktor**
Pls read here: https://hstspreload.appspot.com/

### CODE REVIEW

**Comment by szepeviktor** [comment]
And here: https://coderrr.wordpress.com/2010/12/27/canonical-redirect-pitfalls-with-http-strict-transport-security-and-some-solutions/

File: src/security/strict-transport-security.conf [hunk 1]
```diff
@@ -21,5 +21,5 @@
 # http://blogs.msdn.com/b/ieinternals/archive/2014/08/18/hsts-strict-transport-security-attacks-mitigations-deployment-https.aspx
 
 <IfModule mod_headers.c>
-    Header set Strict-Transport-Security "max-age=16070400; includeSubDomains"
+    Header always set Strict-Transport-Security "max-age=16070400; includeSubDomains"
```
Comment (AD7six) [inline src/security/strict-transport-security.conf]: FWIW (not directly related to this proposed change) `includeSubDomains` is a terrible default, and easily leads to [why can't I access http://foo.example.com](http://serverfault.com/q/656354/108287) confusion.

Comment (szepeviktor) [inline src/security/strict-transport-security.conf]: You could add that as a comment.

Comment (AD7six) [inline src/security/strict-transport-security.conf]: I'll take care of that post-merge. 

Would it be possible for you to add a test for that? I can also do that for you if not.

**Comment by szepeviktor** [comment]
I see no tests for `Strict-Transport-Security`.
Do you mean looking for it in the response headers during a redirected request?

**Comment by szepeviktor** [comment]
I see no redirected requests.

**Comment by szepeviktor** [comment]
Using `RewriteRule` for redirection?

**Comment by alrra** [comment]
@szepeviktor Sorry for the delay on this. 

--

> Header **always** set

@szepeviktor Good catch! :clap:

**Pull request closed without merging**